main.py

```diff
@@ -226,6 +226,7 @@ def update(self, circuit_id):
             log.debug('update result %s %s', exception, 400)
             raise BadRequest(str(exception))
 
+        log.info(f'enable: {enable}, redeploy: {redeploy}')
```

Comment:
I think we should have more information here for the logs, just the flag enabled and redeploy (is it also a flag?) are too generic and not useful for the logs.

Reply:
This one was just for debug, I forgot to remove.